wandb/integration/tensorboard/log.py

```diff
@@ -320,13 +330,21 @@ def _log(
             or timestamp - STEPS["global"]["last_log"] >= RATE_LIMIT_SECONDS
         ):
             if history is None:
+                if wandb.run is None:
+                    raise wandb.Error(
+                        "You must call `wandb.init()` before calling `wandb.tensorflow._log`"
```

Comment:
hmm... i don't think this right to do it here... since it is an internal function that the user doesn't suppose to call... we call it under the hood [here](https://github.com/wandb/client/blob/master/wandb/integration/tensorboard/log.py#L344) and [here](https://github.com/wandb/client/blob/master/wandb/integration/tensorflow/estimator_hook.py#L48) (relevant documentation [estimator-hook](https://docs.wandb.ai/guides/integrations/tensorflow#tensorflow-hook) and [tf-log](https://docs.wandb.ai/guides/integrations/tensorflow#manual-logging)).
We have this message for `tensorflow.log` maybe we can add it in the `estimator hook` constructor, what do you think? here we could have an assertion in that basically checks if history is `None` wandb.run can't be `None`?

Reply:
for the estimator hook: looking at the [begin](https://www.tensorflow.org/api_docs/python/tf/estimator/SessionRunHook#begin) function of the `SessionRunHook`... maybe this is the best place (not sure... also we probably want to move the telemetry logic to the same place)?